homeassistant/components/pulseaudio_loopback/switch.py

```diff
@@ -84,6 +84,11 @@ def __init__(self, host, port, buff_sz, tcp_timeout):
         self._buffer_size = int(buff_sz)
         self._tcp_timeout = int(tcp_timeout)
 
+    @property
+    def state_valid(self):
+        """Return true if module state was updated."""
+        return len(self._current_module_state) > 0
```

Comment:
You could just parse it to a boolean value:
```suggestion
        return bool(self._current_module_state)
```